core/src/glue.rs

```diff
@@ -9,6 +10,7 @@ use {
         translate::translate,
     },
     futures::executor::block_on,
+    futures::stream::{self, StreamExt},
```

Comment:
```suggestion
    futures::stream::{self, StreamExt, TryStreamExt},
```

Please move the ones that are not grouped above thanks !